edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/Program.cs

```diff
@@ -175,8 +175,8 @@ public static async Task<int> MainAsync(IConfiguration configuration)
                         apiVersion = configuration.GetValue<string>(Constants.EdgeletApiVersionVariableName);
                         iothubHostname = configuration.GetValue<string>(Constants.IotHubHostnameVariableName);
                         deviceId = configuration.GetValue<string>(Constants.DeviceIdVariableName);
-                        string networkId = configuration.GetValue<string>(Constants.NetworkIdKey);
                         string proxyImage = configuration.GetValue<string>(K8sConstants.ProxyImageEnvKey);
+                        Option<string> proxyImagePullSecretName = configuration.GetValue<Option<string>>(K8sConstants.ProxyImagePullSecretNameEnvKey);
```

Comment:
Will this work the same as `Option.Maybe(configuration.GetValue<string>(K8sConstants.ProxyImagePullSecretNameEnvKey)`?

Reply:
Yes, it should. If there is no value found it will return default(Option<string>) which is None